server/express-router.md

```diff
@@ -0,0 +1,96 @@
+Previously, we've learnt to separate routes into their own file, by exporting them as functions and requiring them within our `server.js` file:
```

Comment:
Reworded and changed the code example to reflect putting everything in `server.js`